src/components/molecules/Visualizer/Infobox/Frame/index.tsx

```diff
@@ -156,20 +169,22 @@ const TitleFlex = styled(Flex)`
   width: 75%;
 `;
 
-const StyledIcon = styled(Icon)<{ open?: boolean }>`
+const StyledIcon = styled(Icon)<{ open?: boolean; publishedTheme: PublishTheme }>`
```

Comment:
This publishedTheme param is possibly undefined. Please fix it with `publishedTheme?: PublishTheme` and update how it is used for color.

Reply:
`const { publishedTheme } = usePublishedTheme(sceneProperty?.theme);`
It is always defined and has a value, and returned from the hook.

Reply:
Even if param of the hook is undefined, it returns default values for `publishedTheme`.
So it is guaranteed to have a value.

Reply:
If publishedTheme is always defined then update this component's Props (and others that apply). You have it as a conditional prop above.

Reply:
Please explain more, I can't figure out what you mean?
It is not conditional as it always passed and has a value

Reply:
Sorry my bad. I think I was mixing up the sceneProperty prop and the publishedTheme...!